Doc/cmus.txt

```diff
@@ -1230,6 +1230,12 @@ dsp.ao.wav_dir
 	Output directory for libao plugin, default is your home directory.
 	Used only if *dsp.ao.driver* is "wav".
 
+dsp.coreaudio.device
+	Device for Core Audio output. Leave empty for default.
+
+dsp.coreaudio.enable_hog_mode
+	Set true to set the device to hog mode. The default value is false.
```

Comment:
would be nice if you add that to the documentation. I don't think every end user understands the term hog mode.